tuf/api/metadata.py

```diff
@@ -878,20 +878,24 @@ def __init__(
         version: int,
         spec_version: str,
         expires: datetime,
-        meta: Dict[str, MetaFile],
```

Comment:
can you explain this?

Reply:
Yes. I noticed we don't do validation on `Timestamp.meta`.
According to the spec, the value of `Timestamp.meta` called "METAFILES" is:
```
METAFILES is the same as described for the snapshot.json file.
In the case of the timestamp.json file, this MUST only include a
description of the snapshot.json file.
```
That's why in order to include validation for meta no matter if we create objects with `Timestamp.from_dict()`
or with the constructor like `Timestamp()` I moved the meta validation and object creation inside `__init__`.

Reply:
Now with the new discussion about `snapshot_meta` I agree that we shouldn't hurry and add validation.
Will remove the commit.